automation/project/Dependencies.scala

```diff
@@ -53,7 +53,7 @@ object Dependencies {
     "org.scalatest"       %%  "scalatest"     % "3.2.2"   % "test",
     "org.seleniumhq.selenium" % "selenium-java" % "3.8.1" % "test",
     "com.typesafe.scala-logging" %% "scala-logging" % "3.5.0",
-    "org.broadinstitute.dsde"       %% "rawls-model"         % "0.1-14278f08f"
+    "org.broadinstitute.dsde"       %% "rawls-model"         % "0.1-d060dbbc"
```

Comment:
```suggestion
    "org.broadinstitute.dsde"       %% "rawls-model"         % "0.1-8223bdce-SNAP"
```

in case the version got lost in Slack threads, here it is